Rakefile

```diff
@@ -62,7 +62,8 @@ namespace :generate do
 
       # Rails 4+ cannot use a `rails` binstub generated by Bundler
       sh "rm -f #{bindir}/rails"
-      sh "bundle exec rails new ./tmp/example_app --no-rc --skip-javascript --skip-bootsnap -skip-sprockets --skip-git --skip-test-unit --skip-listen --skip-bundle --template=example_app_generator/generate_app.rb"
+      sh "bundle exec rails new ./tmp/example_app --no-rc --skip-javascript --skip-bootsnap -skip-sprockets"\
```

Comment:
There's no space, so this will result in "-skip-sprockets--skip-git" without a separating space.